connection/connection.go

```diff
@@ -104,13 +106,14 @@ func OnConnectionLoss(reconnect func() bool) Option {
 
 // ExitOnConnectionLoss returns callback for OnConnectionLoss() that writes
 // an error to /dev/termination-log and exits.
-func ExitOnConnectionLoss() func() bool {
-	return func() bool {
+func ExitOnConnectionLoss() func(context.Context) bool {
```

Comment:
As the comment says, this is for `OnConnectionLoss`. That option has to be changed accordingly, i.e. take a function with a context parameter.

Reply:
`OnConnectionLoss` still needs to be updated.

Reply:
I've also updated the OnConnectionLoss argument. Please let me know if anything is not as intended.